server/lib/RouteResolver.js

```diff
@@ -11,8 +11,26 @@ const _ = require('lodash');
 let app;
 let fixturesDir;
 
+const validateResponse = function validateResponse(response) {
+  const responseProps = [];
+  const responsePropTypes = ['fixture', 'filePath', 'html', 'json', 'text'];
+
+  Object.keys(response).forEach(key => {
+    responsePropTypes.forEach(propType => {
+      if (key === propType) responseProps.push(response[key]);
+    });
+  });
```

Comment:
@therynamo, this is iterating more than needed. You can limit the number of iterations with an implementation like this:
```js
const payloadKeysPresent = [];
const payloadKeys = ['fixture', 'filePath', 'html', 'json', 'text'];

payloadKeys.forEach(key => {
  if (response[key]) payloadKeysPresent.push(response[key]);
});
```